spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,206 @@
+def find_winner(board)
+    # returns row winner
+    board.each do |row|
+        row_winner = find_winner_in_line(row)
+        if row_winner
+            return row_winner
+        end
+    end
```

Comment:
This code works, but it can also be helpful to use some of the built-in functions in Ruby (which you will see often in other Ruby code).  It might look something like this:

```ruby
row_winner = board
  .map { |row| find_winner_in_line(row) }
  .find { |winner| winner }
return row_winner if row_winner
```

The same thing but in a longer form would look a bit more like this:

```ruby
row_winners = board.map do |row|
  find_winner_in_line(row)
end

first_row_winner = row_winners.find { |winner| winner != false }
if first_row_winner
  return first_row_winner
end
```